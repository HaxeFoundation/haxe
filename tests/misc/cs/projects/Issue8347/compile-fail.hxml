-cp src
Main
fail.NotFirstType
-cs cs-fail
-D no-compilation
