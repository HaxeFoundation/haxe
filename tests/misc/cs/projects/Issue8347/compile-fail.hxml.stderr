src/fail/NotFirstType.hx:8: characters 1-22 : @:cs.assemblyStrict can only be used on the first class of a module
Main.hx:4: characters 1-14 : @:cs.assemblyMeta cannot be used on top level modules
Main.hx:7: characters 1-15 : @:cs.assemblyStrict can only be used on the first class of a module
Main.hx:7: characters 1-15 : @:cs.assemblyStrict cannot be used on top level modules
